regulations/docket.py

```diff
@@ -36,25 +36,30 @@ def get_document_fields(document_id):
         if field['attributeName'] != 'general_comment'
     }
 
-    add_options(fields)
-    add_dependent_options(fields)
+    add_list_options(fields)
+    add_combo_options(fields)
 
     cache.set(document_id, fields)
     return fields
 
 
-def add_options(fields):
-    """Augment fields with options."""
+def add_list_options(fields):
```

Comment:
Since the _type_ is really picklist, can we call it `add_picklist_options` and fix it in the docstring? Again, this is very minor.